examples/otel-logs-with-sourcetypes-splunk/otel-collector-config.yml

```diff
@@ -0,0 +1,72 @@
+receivers:
+    filelog/onefile:
+      include: [ /output/file.log ]
+    filelog/twofile:
+      include: [ /output/file2.log ]
+    filelog/threefolder:
+      include: [ /output3/*.log ]
+
+exporters:
+    splunk_hec/logs:
+        # Splunk HTTP Event Collector token.
+        token: "00000000-0000-0000-0000-0000000000000"
+        # URL to a Splunk instance to send data to.
+        endpoint: "https://splunk:8088/services/collector"
+        # Optional Splunk source: https://docs.splunk.com/Splexicon:Source
+        source: "output"
+        # Default sourcetype. This example overrides sourcetype, so it will never be used.
+        sourcetype: "otel:shouldnotshow"
+        # Splunk index, optional name of the Splunk index targeted.
+        index: "logs"
+        # Maximum HTTP connections to use simultaneously when sending data. Defaults to 100.
+        max_connections: 20
+        # Whether to disable gzip compression over HTTP. Defaults to false.
+        disable_compression: false
+        # HTTP timeout when sending data. Defaults to 10s.
+        timeout: 10s
+        tls:
+          # Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS. Defaults to false.
+          # For this demo, we use a self-signed certificate on the Splunk docker instance, so this flag is set to true.
+          insecure_skip_verify: true
+
+processors:
```

Comment:
Wondering if it might make things easier to understand for readers if the processors block were above the exporters block.

Reply:
moved up